modules/Bio/EnsEMBL/Hive/DBSQL/BeekeeperAdaptor.pm

```diff
@@ -131,5 +131,35 @@ sub reload_beekeeper_is_blocked {
 }
 
 
+=head2 block_all_alive_beekeepers
+
+  Example     : $bk_adaptor->block_all_alive_beekeepers();
+  Description : Set is_blocked for all beekeepers known to the
+                pipeline which haven't died yet. Part of the "shut
+                everything down" feature - as eHive stands we cannot
+                tell other beekeepers to kill their respective active
+                workers (unless said workers happen to belong to the
+                same meadow, in which case we can essentially hijack
+                them) but at least we can prevent them from spawning
+                new workers.
+  Returntype  : none
+  Exception   : none
+  Caller      : beekeeper.pl
+  Status      : Stable
+
+=cut
+
+sub block_all_alive_beekeepers {
+  my ( $self ) = @_;
+
+  my $statement =
+    'UPDATE beekeeper SET is_blocked = 1 WHERE cause_of_death IS NULL';
```

Comment:
In eHive there is no formal maximum line length. I personally find such line breaks are breaking the flow.

Reply:
Personally I like it when lines don't go too far to the right but in this case this wouldn't go _that_ far, concatenated the two.